python/ml4ir/applications/ranking/model/metrics/helpers/metrics_helper.py

```diff
@@ -16,9 +17,83 @@ class RankingConstants:
     DIFF_MRR = "diff_MRR"
     OLD_ACR = "old_ACR"
     NEW_ACR = "new_ACR"
+    OLD_NDCG = "old_NDCG"
+    NEW_NDCG = "NEW_NDCG"
     TTEST_PVALUE_THRESHOLD = 0.1
 
 
+def add_top_graded_relevance(df, label_col, new_col_name):
```

Comment:
@mohazahran Can you make the function name more descriptive? Maybe `add_top_record_column`?

Reply:
done